cmd/libs/go2idl/conversion-gen/generators/conversion.go

```diff
@@ -693,13 +705,22 @@ func (g *genConversion) doStruct(inType, outType *types.Type, sw *generator.Snip
 				sw.Do("out.$.name$ = $.outType|raw$(in.$.name$)\n", args)
 			}
 		case types.Map, types.Slice, types.Pointer:
+			if g.isDirectlyAssignable(m.Type, outMember.Type) {
+				sw.Do("out.$.name$ = in.$.name$\n", args)
```

Comment:
If types are different (i.e. one is an alias for the other), don't we need to cast one on the other?